### PR TITLE
Only addly feed sequencing to inflight and uncommitted lids.

### PR DESCRIPTION
**Pull request by baldersheim**
@toregge PR
Here is a suggestion for discussion.

### CODE REVIEW

File: searchcore/src/vespa/searchcore/proton/server/storeonlyfeedview.cpp [hunk 1]
```diff
@@ -360,31 +369,31 @@ void StoreOnlyFeedView::putSummary(SerialNum serialNum, Lid lid,
...
 
 void StoreOnlyFeedView::putSummary(SerialNum serialNum, Lid lid, Document::SP doc, OnOperationDoneType onDone)
 {
-    _pendingLidTracker.produce(lid);
+    _pendingLidsForDocStore.produce(lid);
 #pragma GCC diagnostic push
 #pragma GCC diagnostic ignored "-Winline" // Avoid spurious inlining warning from GCC related to lambda destructor.
     summaryExecutor().execute(
-            makeLambdaTask([serialNum, doc = std::move(doc), onDone, lid, this] {
+            makeLambdaTask([serialNum, doc = std::move(doc), onDone = std::move(onDone), lid, this] {
```
Comment (toregge) [inline searchcore/src/vespa/searchcore/proton/server/storeonlyfeedview.cpp:383]: std::move is not safe here.

File: searchcore/src/vespa/searchcore/proton/server/storeonlyfeedview.cpp [hunk 2]
```diff
@@ -360,31 +369,31 @@ void StoreOnlyFeedView::putSummary(SerialNum serialNum, Lid lid,
...
+                _pendingLidsForDocStore.consume(lid);
             }));
 #pragma GCC diagnostic pop
 }
 void StoreOnlyFeedView::removeSummary(SerialNum serialNum, Lid lid, OnWriteDoneType onDone) {
-    _pendingLidTracker.produce(lid);
+    _pendingLidsForDocStore.produce(lid);
     summaryExecutor().execute(
-            makeLambdaTask([serialNum, lid, onDone, this] {
+            makeLambdaTask([serialNum, lid, onDone = std::move(onDone), this] {
```
Comment (toregge) [inline searchcore/src/vespa/searchcore/proton/server/storeonlyfeedview.cpp:393]: std::move is not safe here.

File: searchcore/src/vespa/searchcore/proton/server/storeonlyfeedview.cpp (hunk 1 above)
Comment (baldersheim) [reply searchcore/src/vespa/searchcore/proton/server/storeonlyfeedview.cpp]: Fixed

File: searchcore/src/vespa/searchcore/proton/server/storeonlyfeedview.cpp (hunk 2 above)
Comment (baldersheim) [reply searchcore/src/vespa/searchcore/proton/server/storeonlyfeedview.cpp]: Fixed

**Pull request closed without merging**